man/print.data.table.Rd

```diff
@@ -13,6 +13,8 @@
   Key enhancements include automatic output compression of many observations and concise column-wise \code{class} summary.
 
   \code{format_col} and \code{format_list_item} generics provide flexibility for end-users to define custom printing methods for generic classes.
+
+  \code{trunc.char} This feature compresses strings exceeding the \code{datatable.prettyprint.char} limit. Strings longer than this limit have excess characters removed, with "..." added to indicate truncation. Shorter strings remain unchanged.
```

Comment:
trunc.char is the name of an internal function; the benefit of that is we can change it on a moment's notice without worrying about breaking anything.

So, let's stick to describing the _behavior_ without mentioning anything about the _implementation_ (e.g. the name of private functions/algorithms used to accomplish this)